homeassistant/components/elmo_alarm/__init__.py

```diff
@@ -0,0 +1,237 @@
+"""Support for e-connect Elmo alarm system."""
+from collections import namedtuple
+from datetime import timedelta
+import logging
+
+from elmo.api.client import ElmoClient
+from elmo.api.exceptions import PermissionDenied
+from urllib3.exceptions import HTTPError
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_SCAN_INTERVAL,
+    CONF_USERNAME,
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_CUSTOM_BYPASS,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_DISARMED,
+)
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.discovery import async_load_platform
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.event import async_call_later
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "elmo_alarm"
+
+CONF_VENDOR = "vendor"
+
+CONF_STATES = "states"
+CONF_ZONES = "zones"
+
+DEFAULT_SCAN_INTERVAL = timedelta(seconds=5)
+DEFAULT_HOST = "https://connect.elmospa.com"
+
+SIGNAL_ZONE_CHANGED = "elmo_alarm.zone_changed"
+SIGNAL_INPUT_CHANGED = "elmo_alarm.input_changed"
+SIGNAL_ARMING_STATE_CHANGED = "elmo_alarm.arming_state_changed"
```

Comment:
Use f-strings in combination with `DOMAIN`

Reply:
Done.